components/reddit/sources/new-hot-posts-on-a-subreddit/new-hot-posts-on-a-subreddit.js

```diff
@@ -8,9 +8,8 @@ module.exports = {
   name: "New hot posts on a subreddit",
   description:
     "Emits an event each time a new hot post is added to the top 10 items in a subreddit.",
-  version: "0.0.23",
+  version: "0.0.1",
```

Comment:
That reminds me that you'll need to increment the versions of all sources to make sure the new changes are published. Even when you just make changes to the app file and not the sources themselves, this is required.